shell/package.json

```diff
@@ -13,9 +13,9 @@
     },
     "dependencies": {
         "@dhis2/app-adapter": "5.4.2",
-        "@dhis2/app-runtime": "^2.2.2",
+        "@dhis2/app-runtime": "^2.6.1",
         "@dhis2/d2-i18n": "^1.0.5",
-        "@dhis2/ui": "^5.4.2",
+        "@dhis2/ui": "^5.7.2",
```

Comment:
We needed a higher version of `@dhis2/ui` which includes https://github.com/dhis2/ui/pull/300 to avoid an infinite loop when showing more than one alert.